intelmq/bots/parsers/fireeye/parser.py

```diff
@@ -0,0 +1,96 @@
+
+"""
+Fireeye Parser Bot
+Retrieves a base64 encoded JSON-String from raw and converts it into an
+event.
+"""
+from intelmq.lib import utils
+from intelmq.lib.bot import ParserBot
+from intelmq.lib.message import MessageFactory
+from intelmq.lib.utils import base64_decode
+import json
+import xmltodict
```

Comment:
In order to make the basic test suite work, wrap this in a try-except please. And the same logic for the init-method as for the collector applies here as well.